docs/reference/api/resources/paths.rst

```diff
@@ -0,0 +1,55 @@
+App Paths
```

Comment:
```suggestion
Paths
```

There's no more reason for making the sidebar say "App paths" than there is to say "App icons" or "App commands".

Reply:
I'm not sure I agree. 

Icons and Commands are always "for" the app in the sense that the app uses them; however, you can have icons that aren't the icon of the app - most icons aren't the App's icon - so saying "App Icons" would be potentially confusing. In the case of commands, commands are always App based, because they can't be for anything else, so "App Commands" would be redundant. 

However, you *can* have paths that are used by the app, but aren't "app paths" - any user-specified directory, for example, and users will need to use `pathlib.Path` in their code. The Paths object is a subset of very specific paths directly related to the operation of the app, not a generic collection of paths. Mentally, I guess I see this as `app.paths`, since that's the only place where this construct is used; but `app.paths` doesn't make for a good title.